platform/core/src/utils/getNumViewportPanes.ts

```diff
@@ -0,0 +1,17 @@
+import { ViewportGridService } from '../services';
```

Comment:
This should be a member function of ViewportService

Reply:
Sounds good.